sw/device/tests/aes_smoketest.c

```diff
@@ -124,7 +124,7 @@ bool test_main(void) {
 
   // Setup ECB decryption transaction.
   transaction.operation = kDifAesOperationDecrypt;
```

Comment:
@doreis-lowrisc CI is currently failing because `kDifAesOperationEncrypt` and `kDifAesOperationDecrypt` are not properly defined inside `dif_aes.h`. This is now a 2-bit one-hot encoded field. We need:
```c
kDifAesOperationEncrypt = 1;
kDifAesOperationDecrypt = 1 << 1;
```
Related to that, it seems the unit test only tests encryption but not decryption. Should this be corrected as well?

Reply:
Perfect, thanks @doreis-lowrisc , I am going to merge this now.